lib/app/views/problem_test.pg

```diff
@@ -0,0 +1,7 @@
+node :problem_test do
+  node track_id: problem.track_id
+  node slug: problem.slug
+  node test: problem.test_file
+  # deprecated
+  node fresh: false
```

Comment:
We don't need the deprecated node since this is a new file. I've only kept the deprecated one around where the CLI was previously using the result.